rke/k8s_rke_system_images.go

```diff
@@ -1087,13 +1087,13 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			CoreDNS:                   m("coredns/coredns:1.3.1"),
 			CoreDNSAutoscaler:         m("gcr.io/google_containers/cluster-proportional-autoscaler:1.3.0"),
 		},
-		"v1.15.0-rancher1-1": {
+		"v1.15.0-rancher1-2": {
```

Comment:
do we also want to change the default k8s? https://github.com/rancher/kontainer-driver-metadata/blob/master/rke/k8s_version_info.go#L9

Reply:
yes, we do :)